src/routes/e2e/widget/modal/+page.svelte

```diff
@@ -111,6 +111,7 @@
       },
       style: {
         labels: 'floating',
+        showPassword:'checkbox',
```

Comment:
Unfortunately, this change means all test runs will have the checkbox style. What we need to do is programmatically set this only when we want to test the checkbox option. The best way to do that is to use URL query parameters.

Reply:
done